pkg/v1/cli/pluginmanager/test/config.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: config.tanzu.vmware.com/v1alpha1
 clientOptions:
   cli:
     discoverySources:
+    - local:
+        name: default-local
+        path: default
```

Comment:
Because now we are not adding default discovery source add part of `GetClientConfig` adding it in the config file so that existing unit tests works as expected